### PR TITLE
[FIX] travis_install_nightly: Fix the phantomjs patch when the new version of odoo

### PR DESCRIPTION
**Pull request by JesusZapata**
When the `travis/travis_install_nightly` run active the virtual environment of python is activated

And fix the `PHANTOMJS_TESTS` to detect the folder `odoo` also

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 @ruiztulio 
This PR active the virtual environment for python and fix the `phantomjs`

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -91,6 +91,7 @@ echo "x${PHANTOMJS_TESTS}"
 if [ "x${PHANTOMJS_TESTS}" == "x0" ]; then
     # Disable phantom tests
     sed -i "s/'phantomjs'/'disable_phantomjs'/g" ${HOME}/*/openerp/tests/common.py
+    sed -i "s/'phantomjs'/'disable_phantomjs'/g" ${HOME}/*/odoo/tests/common.py
 else
     # Increase default timeout of all phantom tests
     sed -i "s/ login=None, timeout=60, / login=None, timeout=360, /g" ${HOME}/*/openerp/tests/common.py
```
Comment (moylop260) [inline travis/travis_install_nightly]: Use the same patch to odoo here

File: travis/travis_install_nightly [hunk 2]
```diff
@@ -233,4 +234,7 @@ if [ "$PG_LOGS_ENABLE" == "1" ]; then
     python $(dirname ${BASH_SOURCE[0]})/psql_log.py
 fi
 
+# Enable the python version
+source ${REPO_REQUIREMENTS}/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate
```
Comment (moylop260) [inline travis/travis_install_nightly]: Why we need this change here if the installation was on the top?

Comment (moylop260) [reply travis/travis_install_nightly]: Now, MQT shouldn't be very strict to use it just for travis or our image.
I mean, we are working to create a standard library to install package but this change don't help us to the focus.

Maybe we need to change T2D

File: travis/travis_install_nightly [hunk 3]
```diff
@@ -121,6 +121,9 @@ sed -i '/^python-ldap\=\=/d' ${ODOO_PATH}/requirements.txt
 # Use requests with [security] suffix to fix [Errno 111] Connection refused for old python2.7 versions.
 sed -i 's/^requests\=\=/requests[security]\=\=/g' ${ODOO_PATH}/requirements.txt
 
+# Enable the python version
+source ${REPO_REQUIREMENTS}/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate
```
Comment (moylop260) [inline travis/travis_install_nightly]: What about use this sentence from t2d for install script?

**Comment by JesusZapata** [comment]
@moylop260 
I revert the changes related with the `source`, now in this PR only has a fix related with the environment variable `PHANTOMJS_TESTS` and I change the title of this PR in order to get this modification

**Pull request merged**